### DISCUSSION TITLE
Address book on the client does not work.

### DISCUSSION THREAD

**Discussion by micha2021-home** [rustdesk/rustdesk#6531] [Q&A] [answered]
The server runs with the PRO version. The clients can also connect perfectly. Unfortunately I cannot enter login details on the client. There is always a timeout. Ports are all open on the server. Does anyone have an idea what else I can do?

**Comment by DMSnz** [comment]
Have you filled out the API address settings in your client?
if you have this blank I believe (could be wrong here), it uses the ID server address for API address, I also believe API address should be address only no ports listed. But if same as ID server then leaving it blank works a treat for me.

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/docs/en/self-host/client-configuration/#set-api-server

**Comment by gittygittymeowmeow** [comment]
The doc 
https://rustdesk.com/docs/en/self-host/client-configuration/#set-api-server
needs to be updated or perhaps the client or the rustdesk pro console with some instructions.

If I include a port in ver 1.2.3 under Settings>Network>ID/Relay server under API Server for rustdesk pro I get a "semaphore timeout period has expired" when trying to sign into the account/address book from the client.
Instead I put http://<ip or host> without the port and it worked.
For the rustdesk console, you can see there it implies you need to have the port :21114 for the API feild where the text is greyed if not filled. This caused me to include the port in the API feild of my cleint producing the "semaphore timeout period has expired" upon trying to login.
![RD_Console_API](https://github.com/rustdesk/rustdesk/assets/150756848/f9d6ed40-a5d7-435a-835e-21cba551d2f8)
